src/oc-id/config/routes.rb

```diff
@@ -24,6 +24,7 @@
       resources :users, only: :show
 
       get 'me', to: 'users#me'
+      get 'all_users', to: 'users#all_users'
```

Comment:
Is there any reason why we're using this instead of the `index` action of the users controller?